diff --git a/dist/index.js b/dist/index.js
index dcb65059896bd378903f4734ba85c686c214306b..06a04ec82777e3cbf41fc74750a20d3e98067326 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -75,7 +75,7 @@ export const rehypeCodeHook = (options) => {
                 return CONTINUE;
             try {
                 const props = {
-                    code: toText(codeNode, { whitespace: "pre" }),
+                    code: toText(codeNode, { whitespace: 'pre' }).replace(/\r\n/g, '\n'),
                     inline,
                     language,
                     // @ts-expect-error
