diff --git a/dist/index.js b/dist/index.js
index b3bdc9b836f678162cd70de07acac320370c5dff..2e36be10756b4cf2bb796ea5290fdb2e3d1da9bb 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -6,6 +6,7 @@ export const rehypeCodeHookImg = (options) => {
         defaults = { ...pluginDefaults, ...defaults };
         // @ts-expect-error
         return rehypeCodeHook({
+            cache,
             salt: defaults,
             language,
             code: ({ code, meta }) => {
